templates/sway-script-test-rs/template/tests/harness.rs

```diff
@@ -0,0 +1,36 @@
+use fuels::prelude::*;
+
+// Load abi from json
+abigen!(Script(
+    name = "MyScript",
+    abi = "out/debug/{{project-name}}-abi.json"
+));
+
+async fn get_script_instance() -> MyScript<WalletUnlocked> {
+    // Launch a local network
+    let mut wallets = launch_custom_provider_and_get_wallets(
+        WalletsConfig::new(
+            Some(1),             /* Single wallet */
+            Some(1),             /* Single coin (UTXO) */
+            Some(1_000_000_000), /* Amount per coin */
+        ),
+        None,
+        None,
+    )
+    .await;
+    let wallet = wallets.pop().unwrap();
+
+    let bin_path = "./out/debug/{{project-name}}.bin";
+
+    let instance = MyScript::new(wallet.clone(), bin_path);
+
+    instance
+}
+
+#[tokio::test]
+async fn can_get_script_instance() {
+    let instance = get_script_instance().await;
+    let _response = instance.main().call().await;
+
+    // Now you have an instance of your script
```

Comment:
It would be good to add some example assertions to show common things you might do in a sway integration test.

Reply:
@calldelegation Would you mind adding the above this so we can merge this PR?

Reply:
Yes of course! Ill put it on the the top of my list when I get back :)